javascript/javascript1/week4/Arrays-06.js

```diff
@@ -0,0 +1,47 @@
+// 1. Fav foods
+
+const favFoods = [
+  "Dates",
+  "Olives",
+  "Figs",
+  "Salt",
+  "Vinegar",
+  "Black cumin",
+  "Honey",
+];
+for (const food of favFoods) {
+  console.log(food);
+}
+
+// 2. Sum of numbers
+
+function sumArray(numbers) {
+  let sum = 0;
+  for (const num of numbers) {
+    sum = sum + num;
```

Comment:
This can be also written as
```suggestion
    sum += num;
```